docs/end-user/cloud-services.md

```diff
@@ -2,4 +2,132 @@
 title:  Integrate Cloud Service
 ---
 
-WIP
\ No newline at end of file
+Cloud-oriented development is now becoming the norm, there is an urgent need to integrate cloud resources from different sources and types. Whether it is the most basic object storage, cloud database, or load balancing, it is all faced with the challenges of hybrid cloud, multi-cloud and other complex environments, and KubeVela is perfect for the needs.
+
+KubeVela efficiently and securely integrates different types of cloud resources through resource binding capabilities in cloud resource Components and Traits. At present, you can directly use the default components of Ali Cloud Kubernetes(ACK), Ali Cloud Object Storage Service (OSS) and Ali Cloud Relational Database Service (RDS). At the same time, more new cloud resources will gradually become the default option under the support of the community in the future, so that you can use cloud resources of various manufacturers in a standardized and unified way.
```

Comment:
```suggestion
KubeVela efficiently and securely integrates different types of cloud resources through resource binding capabilities in cloud resource Components and Traits. At present, you can directly use the default components of AliCloud Kubernetes(ACK), AliCloud Object Storage Service (OSS) and AliCloud Relational Database Service (RDS). At the same time, more new cloud resources will gradually become the default option under the support of the community in the future, so that you can use cloud resources of various manufacturers in a standardized and unified way.
```